crates/aptos/src/move_tool/stored_package.rs

```diff
@@ -149,6 +179,18 @@ impl<'a> CachedPackageMetadata<'a> {
         Ok(())
     }
 
+    pub fn save_bytecode_to_disk(
+        &self,
+        path: &Path,
+        module_name: &str,
+        bytecode: &[u8],
+    ) -> anyhow::Result<()> {
+        let bytecode_dir = path.join(CompiledPackageLayout::CompiledModules.path());
+        fs::create_dir_all(&bytecode_dir)?;
+        fs::write(bytecode_dir.join(format!("{}.mv", module_name)), bytecode)?;
+        Ok(())
+    }
```

Comment:
I think there's a command for this somewhere in the utils.

`create_dir_if_not_exist(dir: &Path)`

And I think there is also one for the individual files.

Could clean up afterwards.

Reply:
I followed the pattern used multiple times in this module, if we clean it up, all should be changed.